src/main/resources/db/changelog/changes/07-create-books-categories-table.yaml

```diff
@@ -9,11 +9,15 @@ databaseChangeLog:
               - column:
                   constraints:
                     nullable: false
+                    foreignKeyName: fk_books_categories_books
```

Comment:
pls do not update existing Liquibase migrations


why not creating a separate migraton for that?

you could've used `addForeignKeyConstraint` there

Reply:
Okay, I'll take that into account. Now there’s probably no point in adding it separately.